app/src/test/java/tech/ula/utils/ServerUtilityTest.kt

```diff
@@ -23,15 +23,15 @@ class ServerUtilityTest {
 
```

Comment:
This should be removed since the `LogUtility` class was removed

Reply:
All references to LogUtility in this test file should be removed, that may be the cause of CI failing.

Reply:
Yep, that should fix it.